src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchPropertiesSpec.groovy

```diff
@@ -188,26 +188,4 @@ class SwitchPropertiesSpec extends HealthCheckSpecification {
         String server42IslRtt
         Pattern description = null
     }
-
-    @Tags([TOPOLOGY_DEPENDENT, SMOKE_SWITCHES])
```

Comment:
is it an obsolete test?

Reply:
> is it an obsolete test?

Both OVS and modern switches support VXLAN. We just don't have a lab to run this test. I assumed that there's no sense to continue to maintain tests which we don't have where to run.